File: Proyecto Python/Python/Ejercicio11.py
pares= 0
numeros_5= 0
total_numeros= 0

#Se usa while que se ejecuta hasta que se cumpla alguna condición
while pares < 100 and numeros_5 < 80:
    num= int(input("Ingrese un número positivo: "))

    if num < 0:
        print("Solo números positivos")
        continue

    total_numeros+= 1

    if num % 2 == 0:
        pares+= 1

    if "5" in str(num):
        numeros_5+= 1

print(f"Total de números ingresados: {total_numeros}")
print(f"Total de números pares: {pares}")
print(f"Total de números que contienen 5: {numeros_5}")

File: Proyecto Python/src/tallerb/Ejercicio11.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Main.java to edit this template
 */
package tallerb;

import java.util.Scanner;

/**
 *
 * @author javier
 */
public class Ejercicio11 {

    /**
     * @param args the command line arguments
     */
    public static void main(String[] args) {
        Scanner entrada=new Scanner(System.in);
        
        int totalleidos=0;
        int pares=0;
        int numeroscon5=0;
        
        while(pares < 100 && numeroscon5 <80){
            System.out.print("Ingrese un número positivo: ");
            int num=entrada.nextInt();
            
            if(num < 0){
                System.out.println("Solo se permiten los números positivos.");
                continue;
            }
            totalleidos++;
            
            if(num % 2 == 0){
                pares++;
            }
            
            int numeroaux =num;
            
            while(numeroaux > 0){
                if(numeroaux % 10 == 5){
                    numeroscon5++;
                    break;
                }
            
                numeroaux/=10;
            }
        }
            
        System.out.println("Total de números leídos: "+totalleidos);
        System.out.println("Total de números pares encontrados: "+pares);
        System.out.println("Total de números con digito 5: "+numeroscon5);
            
    }        
      
}
